public/style/carte.css

```diff
@@ -14,6 +14,7 @@ body {
     text-rendering: optimizelegibility;
     margin: 0;
     padding: 0;
+    background-color:black!important;
```

Comment:
what is "!important" ?

Reply:
by default the background is transparent, so my page's background color is white also the links are too. To make it seems better i used "!important"